src/reducers/ui.js

```diff
@@ -337,6 +339,17 @@ export default (state = EMPTY_UI, action) => {
                 },
             }
         }
+        case REMOVE_UI_REPETITION: {
+            const {
+                [action.value]: removedProperty,
```

Comment:
I only find one instance of "removedProperty" in the code base. Is that expected?

Reply:
nvm, sorry this is just renaming...